test/specs/buffer-with-count.coffee

```diff
@@ -0,0 +1,107 @@
+{stream, prop, send, Kefir} = require('../test-helpers.coffee')
+
+describe 'bufferWithCount', ->
+
+  describe 'stream', ->
+
+    it 'should return stream', ->
+      expect(stream().bufferWithCount(1)).toBeStream()
+
+    it 'should activate/deactivate source', ->
+      a = stream()
+      expect(a.bufferWithCount(1)).toActivate(a)
+
+    it 'should be ended if source was ended', ->
+      expect(send(stream(), ['<end>']).bufferWithCount(1)).toEmit 
+      []
+      ['<end:current>']
```

Comment:
sorry, no language should have let me do that :)

what I meant to do (and what i would expect to happen) is this:

```coffee
 it 'should be ended if source was ended', ->
    expect(send(stream(), ['<end>']).bufferWithCount(1)).toEmit([
        [],
        '<end:current>'
      ])
```

what i see now, and what passes the tests (now that i am passing in the options correctly), is this:

```coffee
    it 'should be ended if source was ended', ->
      expect(send(stream(), ['<end>']).bufferWithCount(1)).toEmit([
        { current: [] },
        '<end:current>'
      ])
```

i am confused about the `{ current }` value. my understanding was that only properties have the notion of current values, but here we're passing in a stream?